lm_eval/tasks/diabla.py

```diff
@@ -69,3 +73,150 @@ def invalid_doc_for_prompt(self, doc) -> bool:
         if len(self.doc_to_target(doc)) == 0 or self.doc_to_target(doc)[0] == "":
             return True
         return False
+
+
+class DiaBLa_1_shot_context(PromptSourceTask):
+
```

Comment:
Add a 1 sentence class docstring to make others aware of what's different here from the vanilla `DiaBla` class.